tests/fast_tests/test_files/fig8.json

```diff
@@ -0,0 +1,151 @@
+{
+    "env": {
+        "additional_params": {
+            "max_accel": 3,
+            "max_decel": 3,
+            "sort_vehicles": false,
+            "target_velocity": 20
+        },
+        "clip_actions": true,
+        "evaluate": false,
+        "horizon": 1500,
+        "sims_per_step": 1,
+        "warmup_steps": 0
+    },
+    "env_name": "AccelEnv",
+    "exp_tag": "figure_eight_0",
+    "initial": {
+        "additional_params": {},
+        "bunching": 0,
+        "edges_distribution": "all",
+        "lanes_distribution": Infinity,
```

Comment:
Is this a syntax error?

Reply:
Oh no, it's how json stores inf numbers. It doesn't cause any bug though, don't worry.